pymatgen/core/surface.py

```diff
@@ -66,37 +69,35 @@ class Slab(Structure):
 
     Attributes:
         miller_index (tuple): Miller index of plane parallel to surface.
-        scale_factor (float): Final computed scale factor that brings the parent cell to the surface cell.
```

Comment:
This is super confusing. Need a closer look. `scale_factor` seems to be a `np.ndarray` not a `float`